bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -123,6 +135,26 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList',
         tobeRemoved.remove();
         chosen.trigger('chosen:updated');
     },
+    /** @method disableOption disables an options where value mathces id
+     *   @param { String } id string to find
+     */
+    disableOption: function ( id ) {
```

Comment:
Maybe rename this disableOptions and take an array as parameter. Then chosen doesn't need to trigger an update for each indicator (that could be hundreds of times in a row)